slicer_cli_web/models/docker_image.py

```diff
@@ -185,14 +185,16 @@ def saveImage(name, cli_dict, docker_image, user, baseFolder):
         folderModel.setMetadata(tag, labels)
 
         folderModel.setMetadata(tag, dict(slicerCLIType='tag', slicerCLIRestPath=restPath))
-        folderModel.clean(tag)
+        existingItems = {item['name']: item for item in list(folderModel.childItems(tag, filters={'meta.slicerCLIType': 'task'}))}
```

Comment:
I don't think you need to wrap the `childItems` call in `list()` as the dictionary comprehension will iterate through it as it.